src/types.ts

```diff
@@ -28,7 +30,8 @@ export enum IncomingActionType {
   SCROLL_POSITION = 'scroll_position',
   TRACK_SUMMARY = 'track_summary',
   ZMENU_CREATE = 'zmenu',
-  ZMENU_REPOSITION = 'update_zmenu_position'
+  ZMENU_REPOSITION = 'update_zmenu_position',
```

Comment:
Not related to the scope to this PR; but could you check with Dan if he thinks this action is going to be useful at all?